lib/mongoidable/current_ability.rb

```diff
@@ -8,10 +8,11 @@ module Mongoidable
   #   own static class abilities (including base class abilities)
   #   own instance abilities
   module CurrentAbility
-    def current_ability
+    attr_reader :parent_model
+    def current_ability(parent = nil)
       abilities = Mongoidable::Abilities.new(mongoidable_identity)
```

Comment:
Why not set `@parent_model` here instead of passing it in to `add_ancestral_abilities`?